### ISSUE TITLE
Incorrectly flags boolean props with no value

### ISSUE THREAD

**Issue by dixhuit** [vuetifyjs/eslint-plugin-vuetify#81]
As I understand it, Vue let's you just specify just the prop name and it will assume you're setting the value to `true`. E.g:

```
<v-progress-linear
  indeterminate
/>
```
Is the same as:

```
<v-progress-linear
  :indeterminate="true"
/>
```

But eslint-plugin-vuetify flags the former example as incorrect even though it's fine and it works.

> indeterminate requires value

**Comment by KaelWD** [comment]
That isn't from eslint-plugin-vuetify.
